docs/projects/tad/reporting-standard/latest.md

```diff
@@ -73,10 +73,12 @@ A `system_card` contains the following information.
     4. `author` (OPTIONAL, string). Name of person that initiated the transformations.
 
 3. `name` (OPTIONAL, string). Name used to describe the system.
-4. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+4. `instruments` (OPTIONAL, list[str]). List of Uniform Resource Names of instruments from
```

Comment:
@ChristopherSpelt I would propose to make this a list of objects i.s.o. of a list of strings. If we make this change we could include additional information such as `instrument_version` and `required`.